meta-mender-tegra/meta-mender-tegra-jetpack5/scripts/test/README.md

```diff
@@ -0,0 +1,54 @@
+# Tegra Mender Torture Tests
```

Comment:
Probably should just delete this directory for now, won't work with Jetpack 5 as-is.  If we do add Jetpack 5 support we should probably find some way to share between 4 and 5 directories.